internals/plan/plan.go

```diff
@@ -560,15 +561,12 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 			case ReplaceOverride:
 				combined.Services[name] = service.Copy()
 			case UnknownOverride:
-				return nil, &FormatError{
-					Message: fmt.Sprintf(`layer %q must define "override" for service %q`,
-						layer.Label, service.Name),
-				}
+				errorMessages = append(errorMessages, fmt.Sprintf(`layer %q must define "override" for service %q`,
+					layer.Label, service.Name))
```

Comment:
I think this part works fine as is, because if override is missing/invalid, we won't add to the `combined.Services` map, so it won't be checked below.